src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,35 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
 
     public void getStatistic(String fromFileName, String toFileName) {
+        int supply = 0;
+        int buy = 0;
+        StringBuilder builder = new StringBuilder();
+        try (BufferedReader bufferedReader = new BufferedReader(new FileReader(fromFileName))) {
+            for (String line = bufferedReader.readLine(); line != null; line = bufferedReader.readLine()) {
+                if (line.substring(0, line.indexOf(",")).equals("supply")) {
+                    supply += Integer.parseInt(line.substring(line.indexOf(",") + 1));
+                } else if (line.substring(0, line.indexOf(",")).equals("buy")) {
```

Comment:
let's move this `line.substring(0, line.indexOf(",") + 1)` into a local variable
and make `,` a constant field